workers/unity/Assets/Fps/Scripts/SetupLogic/WorkerConnectorBase.cs

```diff
@@ -61,5 +83,18 @@ protected virtual IEnumerator LoadWorld()
                 Worker.LogDispatcher,
                 this);
         }
+
+        protected void LoadDevAuthToken()
+        {
+            var textAsset = Resources.Load<TextAsset>("DevAuthToken");
```

Comment:
Hardcoded path, while the save location is configurable

Reply:
we do specify in the gdk tools config window that you need to specify the root of a resources folder exactly for that reason.
Resources.Load will check for that file in the root of all Resources folder. So that will work